### ISSUE TITLE
Unsure if running native or container?

### ISSUE THREAD

**Issue by cdrage** [containers/ramalama#74]
I purposely have podman machine not running, however ramalama still works?

```
▶ podman ps
Cannot connect to Podman. Please verify your connection to the Linux system using `podman system connection list`, or try `podman machine init` and `podman machine start` to manage a new Linux VM
Error: unable to connect to Podman socket: failed to connect: dial tcp 127.0.0.1:50279: connect: connection refused

~                                                                             ⍉
▶ ramalama run llama3
system

You are a helpful assistant


>
```

i'm confused in the docs if this is using native python on the host system or still being ran through a container?

**Comment by ericcurtin** [comment]
On macOS native we don't use containers because containers don't exist in macOS, we should document this somewhere...

But... We can still pull models as OCI artefacts on macOS... So one can use container transport mechanisms...

We will also support GPU acceleration using @slp 's mesa patches and podman-machine too on macOS

We also plan on adding a --no-container option on Linux if someone wants to run outside a container...

I'd recommend most should run within a container, keeping your base system clean and use a tested runtime environment via a container... The Containerfile isn't so complex now, but once we work on the auto-detect GPU stuff, it will get more complex...

**Comment by rhatdan** [comment]
Yes the goal on MAC is to use containers if podman and podman-machine are present. (Or Docker and Docker-machine) but if not, then it will run natively.
